### ISSUE TITLE
should work with multiline if statements that do not include then

### ISSUE THREAD

**Issue by georgejecook** [TwitchBronBron/brightscript-formatter#13]
```	if url.instr(".m3u8") >= 0
		return "hls"
	else if url.instr(".mpd") >= 0
		return "dash"
	else
		return "hls"
	end if```
should be formatted correctly; instead if without then is ignored/badly formatted.

**Comment by georgejecook** [comment]
this is addressed in https://github.com/TwitchBronBron/brightscript-formatter/pull/14

**Comment by evgygor** [comment]
`Then` strictly separates the condition from action. In lot of cases, it's hard to see where complicated condition block ends and action starts.
Just use then and code always will be formatted well.

**Comment by georgejecook** [comment]
It's valid brightscript. It should be permitted. I'm not going to change
how I code, my codebase or a client's codebases just because a formatter
has a bug.

**Comment by evgygor** [comment]
Btw, the PR failed due to code coverage issues, You need to add the unit tests

**Comment by georgejecook** [comment]
Okay. I shall do that.

Sorry I did I didnt see that. I was focused. On the vscode ide.. I actually
have this codefile in that project on my local machine and just copied the
code across. Was late and I was sloppy.

**Comment by TwitchBronBron** [comment]
I agree, if it's valid brightscript, it should be supported. I just didn't have the motivation to fix it because I always explicitly use the word `then`. This is more than welcome! 

And yes, please include unit tests that both validate that this would, and meets the code coverage requirements. 

Thanks for your contribution!

**Comment by georgejecook** [comment]
blocked by https://github.com/TwitchBronBron/brightscript-formatter/issues/15
